src/pages/Detail/components/BookingDatePicker.js

```diff
@@ -0,0 +1,257 @@
+
+  const months = [
+    '1월',
+    '2월',
+    '3월',
+    '4월',
+    '5월',
+    '6월',
+    '7월',
+    '8월',
+    '9월',
+    '10월',
+    '11월',
+    '12월',
+  ];
+
+  return (
+    <DatePikerModal>
+      <ModalBg onClick={handleModal} />
+      <ModalInner>
+        <Calendar
+          value={bookingData}
+          numberOfMonths={2}
+          onChange={values => {
+            setBookingData(values);
+          }}
+          weekDays={weekDays}
+          months={months}
+          showOtherDays
+          range
+          hideYear
+          disableMonthPicker
+          disableYearPicker
+        >
+          <ClearDatePiker onClick={clearDatePiker}>
+            선택날짜 지우기
+          </ClearDatePiker>
+        </Calendar>
+      </ModalInner>
+    </DatePikerModal>
+  );
+};
+
+export default BookingDatePicker;
+
+const DatePikerModal = styled.div`
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 100000;
+`;
+
+const ModalBg = styled.div`
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  z-index: 200;
+`;
+
+const ModalInner = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 1130px;
+  height: 420px;
+  background-color: #fff;
+  border-radius: 10px;
+  padding: 40px 40px;
+  overflow: hidden;
+  z-index: 10000000;
+
+  animation: modal-animation 300ms;
+  @keyframes modal-animation {
+    from {
+      opacity: 0;
+    }
+
+    to {
+      opacity: 1;
+    }
+  }
+
+  .rmdp-wrapper {
+    box-shadow: none;
+    width: 100%;
+  }
+
+  .rmdp-top-class {
+    width: 100%;
+  }
+
+  .rmdp-header-values {
+    span {
+      padding-right: 460px;
+      font-size: 20px;
+      font-weight: 700;
+    }
+  }
+
+  .rmdp-calendar {
+    width: 100%;
+  }
+
+  .rmdp-deactive {
+    .sd {
+      color: #c9c9c9 !important;
+    }
+  }
+
+  .rmdp-day-picker {
+    width: 100%;
+
+    div {
+      width: 100%;
+
+      div {
+        color: black;
+        font-size: 16px;
+        font-weight: 500;
+      }
+    }
+  }
+
+  .rmdp-arrow-container {
+    display: none !important;
+  }
+
+  .rmdp-week {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    .rmdp-week-day {
+      color: #000;
+    }
+
+    .rmdp-day {
+      height: 40px;
+
+      &.rmdp-range {
+        background-color: #000;
+        color: #000;
+
+        span {
+          color: #fff;
+        }
+      }
+
+      &.rmdp-range.start {
+        border-bottom-left-radius: 40px;
+        border-top-left-radius: 40px;
+      }
+
+      &.rmdp-today {
+        span {
+          background-color: gray;
+          color: #fff;
+          border-bottom-left-radius: 40px;
+          border-top-left-radius: 40px;
+          border-bottom-right-radius: 40px;
+          border-top-right-radius: 40px;
```

Comment:
```suggestion
border-radius:40px;
```
4방향 모두 같다면 한줄로 줄일 수 있습니다